scilpy/samplingscheme/multiple_shell_energy.py

```diff
@@ -0,0 +1,258 @@
+################################################
```

Comment:
Once https://github.com/nipy/dipy/pull/1711/files is merged in Dipy, we should use Dipy's version to have less code redundancy.

@arnaudbore can you open an issue here to track it?

Reply:
Done